### DISCUSSION TITLE
default 2 node cluster No valid partition issue

### DISCUSSION THREAD

**Discussion by rathken** [vesoft-inc/nebula#5505] [Q&A]
Hi, 

I am running a really simple test system with 2 nodes, using the default configs I am getting:

192.168.1.21 | 9779 | ONLINE | 0 | No valid partition | No valid partition |  
192.168.1.22 | 9779 | ONLINE | 1 | storage_client_test:1 | storage_client_test:1

And I don't understand why or how to resolve.  

The data directories on the 2 servers also show so very different sizes.

**Comment by QingZ11** [comment]
The number of replica nodes deployed in Nebula needs to be an odd number so that raft can elect a leader (the majority wins). You can take a brief look at the raft introduction document to understand the election mechanism.
